### PR TITLE
Remove JS context from SQL files

### PR DESCRIPTION
**Pull request by Ekrekr**
Update uniqueKey field description while here.

### CODE REVIEW

File: core/compilers.ts [hunk 1]
```diff
@@ -37,10 +37,7 @@ export function compile(code: string, path: string): string {
   }
   if (Path.fileExtension(path) === "sql") {
     const { sql } = extractSqlxParts(SyntaxTreeNode.create(code));
```
Comment (BenBirt) [inline core/compilers.ts:39]: i think this is pretty heavyweight, just for the sake of some escaping. i think you should just

```
const escapedCode = code.replace('`', '\\`')}.replace('${', '\\${');
return `exports.query = `${escapedCode}`;`;
```

**Review by BenBirt — CHANGES_REQUESTED** [review]
Surely to be semantically equivalent you should also add `dependencyTargets: ...` in the `actions.yaml` for `extreme_weather_programming` actions?

File: core/compilers.ts (hunk 1 above)
Comment (Ekrekr) [reply core/compilers.ts:39]: Wait, why `.replace('${', '\\${')`? I thought the chat meant no JS in the SQL at all, not just no contextables

**Comment by Ekrekr** [comment]
> Surely to be semantically equivalent you should also add `dependencyTargets: ...` in the `actions.yaml` for `extreme_weather_programming` actions?

Done

**Review by BenBirt — APPROVED** [review]

**Pull request merged**